src/testWithSpringBoot_2_7/java/org/openrewrite/java/spring/boot2/AuthorizeHttpRequestsTest.java

```diff
@@ -0,0 +1,288 @@
+              import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+              import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+
+              @EnableWebSecurity
+              public class SecurityConfig extends WebSecurityConfigurerAdapter {
+
+                  @Override
+                  protected void configure(HttpSecurity http) throws Exception {
+                      /*TODO: replace removed '.accessDecisionManager(null);' with appropriate call to 'access(AuthorizationManager)' after antMatcher(...) call etc.*/
+                      http
+                          .authorizeHttpRequests()
+                              .antMatchers("/blog/**").permitAll()
+                              .anyRequest().authenticated();
+                  }
+              }
+                             """
+          )
+        );
+    }
+
+    @Test
+    void accessDecisionManager() {
+        //language=java
+        rewriteRun(
+          java(
+            """
+              import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+              import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+              import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+
+              @EnableWebSecurity
+              public class SecurityConfig extends WebSecurityConfigurerAdapter {
+
+                  @Override
+                  protected void configure(HttpSecurity http) throws Exception {
+                      http
+                          .authorizeRequests()
+                              .accessDecisionManager(null)
+                              .antMatchers("/blog/**").permitAll()
+                              .anyRequest().authenticated();
+                  }
+              }
+                            """,
+            """
+              import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+              import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+              import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+
+              @EnableWebSecurity
+              public class SecurityConfig extends WebSecurityConfigurerAdapter {
+
+                  @Override
+                  protected void configure(HttpSecurity http) throws Exception {
+                      http
+                          .authorizeHttpRequests()
+                              /*TODO: replace removed '.accessDecisionManager(null);' with appropriate call to 'access(AuthorizationManager)' after antMatcher(...) call etc.*/
+                              .antMatchers("/blog/**").permitAll()
+                              .anyRequest().authenticated();
+                  }
+              }
+                             """
```

Comment:
These closing quotes should only be indented two spaces further than the opening quotes. But it can easily be fixed after the fact with this recipe for just such a purpose:

https://public.moderne.io/recipes/org.openrewrite.java.recipes.SourceSpecTextBlockIndentation